lib/flutter_app_version_checker.dart

```diff
@@ -40,20 +46,21 @@ class AppVersionChecker {
           return await _checkPlayStore(_currentVersion, _packageName);
       }
     } else if (Platform.isIOS) {
-      return await _checkAppleStore(_currentVersion, _packageName);
+      return await _checkAppleStore(
+          _currentVersion, _packageName, _countryCodes);
     } else {
       return AppCheckerResult(_currentVersion, null, "",
           'The target platform "${Platform.operatingSystem}" is not yet supported by this package.');
     }
   }
 
-  Future<AppCheckerResult> _checkAppleStore(
-      String currentVersion, String packageName) async {
+  Future<AppCheckerResult> _checkAppleStore(String currentVersion,
+      String packageName, List<String> countryCodes) async {
     String? errorMsg;
     String? newVersion;
     String? url;
-    var uri =
-        Uri.https("itunes.apple.com", "/lookup", {"bundleId": packageName});
+    var uri = Uri.https("itunes.apple.com", "/lookup",
+        {"bundleId": packageName, "country": countryCodes});
```

Comment:
Shouldn't this be `countryCodes.join(',')`?

Reply:
No, the iTunes API expect many `country=<code>` parameters rather than `country=<code,code,code...>`